posts/2022-01-20-beta-release-post.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+blog_description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+
+[#allbeta]
```

Comment:
[#allbeta]
does this anchor belong here? What links to the top of the doc? Seems like it should be on the try it now section and linked to from here... but Im not sure